docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -193,14 +193,16 @@ This section describes optional configuration options, particularly for ensuring
 * `pbxes` - a comma separated list of IP address that Bono considers to be PBXes that are incapable of registering. Non-REGISTER requests from these addresses are passed upstream to Sprout with a `Proxy-Authorization` header. It is strongly recommended that Sprout's `non_register_authentication` option is set to `if_proxy_authorization_present` so that the request will be challenged. Bono also permits requests to these addresses from the core to pass through it.
 * `pbx_service_route` - the SIP URI to which Bono routes originating calls from non-registering PBXes (which are identified by the `pbxes` option). This is used to route requests directly to the S-CSCF rather than going via an I-CSCF (which could change the route header and prevent the S-CSCF from processing the request properly). This URI is used verbatim and should almost always include the `lr`, `orig`, and `auto-reg` parameters. If this option is not specified, the requests are routed to the address specified by the `upstream_hostname` and `upstream_port` options.
     * e.g. `sip:sprout.example.com:5054;transport=tcp;lr;orig;auto-reg`
-* `non_register_authentication` - controls when Sprout will challenge a non-REGISTER request using SIP Proxy-Authentication. Possible values are `never` (meaning Sprout will never challenge) or `if_proxy_authorization_present` (meaning Sprout will only challenge requests that have a Proxy-Authorization header).
+* `non_register_authentication` - controls when Sprout will challenge a non-REGISTER request using SIP Proxy-Authentication. This option is a comma separated list that may contain the values listed below (e.g. `non_register_authentication=if_proxy_authorization_present,initial_request_from_req_digest_endpoint`):
+  * `if_proxy_authorization_present`: Sprout will authenticate requests that have a Proxy-Authorization header.
+  * `initial_req_from_reg_digest_endpoint`: Sprout will authenticate requests from registered endpoints that use the SIP digest authentication scheme.
```

Comment:
On line 198 you say "initial_req_from_reg_digest_endpoint"
This is different from the ex on line 196 where you say "initial_request_from_req_digest_endpoint"